### PR TITLE
Fix/update GitHub codepen onboarding

### PR DESCRIPTION
**Pull request by QuincyLarson**
GitHub and CodePen changed their on-boarding, so I created new gifs and instructions to reflect this. 

I also moved some awkward br elements from challenge descriptions.

I also added special copy that told people who already had GitHub / CodePen accounts how to proceed, and an explanation for why we don't have a skip button (since that is a feature we don't want to implement because people will compulsively click it and we'll spend all day manually telling people how to set up GitHub accounts, etc.)

The wiki has changed significantly. I don't think we need the wiki challenge any more.

I gave the tribute project a creation date so it will be marked as new.

I also removed the jQuery examples from the CodePen onboarding flow, since campers won't have completed jQuery by the time they reach that.

### CODE REVIEW

**Comment by tinkbl67** [comment]
I would like to know what these codes and git hub is used for exactly.  I see my hub on it..but he says it nothing..but I know he's hiding info

File: seed/challenges/01-front-end-development-certification/basic-ziplines.json [hunk 1]
```diff
@@ -23,14 +23,14 @@
...
+          "http://i.imgur.com/G1saeDt.gif",
           "A gif showing how to create a Codepen account.",
-          "For our front end project challenges, we'll use a popular browser-based code editor called CodePen. Open CodePen and click \"Sign up\" in the upper right hand corner, then scroll down to the free plan and click \"Sign up\" again. Click the \"Use info from GitHub button\", then add your email address and create a password. Click the \"Sign up\" button. Then in the upper right hand corner, click \"New pen\".",
-          "http://codepen.io"
+          "For our front end project challenges, we'll use a popular browser-based code editor called CodePen. Click the \"Open link in new tab\" button below to open CodePen's signup page. Fill out the form and click \"Sign up\". <br><div class=\"small\">Note: If you already have a CodePen account, you can skip this step by clicking \"Open link in new tab\", closing the new tab that opens, then clicking \"go to my next step\". We removed our \"skip step\" button because many people would just click it repeatedly without doing these important steps.</div>",
+          "http://codepen.io/signup/free"
         ],
```
Comment (BerkeleyTrue) [inline seed/challenges/01-front-end-development-certification/basic-ziplines.json]: This URL does not take you to a free signup page if you're already signed in. Not sure if that is a problem.

Comment (QuincyLarson) [inline seed/challenges/01-front-end-development-certification/basic-ziplines.json]: The alternative would be for everyone to have to go through an additional step (codepen.io/signup and then choose free, which is practically hidden on the page). I've chosen to bypass this. I suspect most people who already have CodePen accounts will skip this step anyway.

**Comment by maaxx2004** [comment]
The info make more explore to real git and code-pen the best flatform pro developer as well as AWS provide.

**Comment by KidCamp** [comment]
I'm having trouble signing up for codepen. Every time i click the signup link i get sent to a error page

**Comment by texas2010** [comment]
@KidCamp, try this link. http://codepen.io/
if you have a problem with codepen account. please talk to codepen support.

Have a nice day!

**Pull request merged**